portable/NetworkInterface/WinPCap/NetworkInterface.c

```diff
@@ -521,18 +521,31 @@ static BaseType_t xPacketBouncedBack( const uint8_t * pucBuffer )
 {
     EthernetHeader_t * pxEtherHeader;
     BaseType_t xResult;
+    static BaseType_t xHasPrinted = pdFALSE;
 
     pxEtherHeader = ( EthernetHeader_t * ) pucBuffer;
 
+    /* Sometimes, packets are bounced back by the driver and we need not process them. Check
+     * whether this packet is one such packet. */
     if( memcmp( ucMACAddress, pxEtherHeader->xSourceAddress.ucBytes, ipMAC_ADDRESS_LENGTH_BYTES ) == 0 )
     {
-        FreeRTOS_printf( ( "Bounced back: %02x:%02x:%02x:%02x:%02x:%02x\n",
-                           pxEtherHeader->xSourceAddress.ucBytes[ 0 ],
-                           pxEtherHeader->xSourceAddress.ucBytes[ 1 ],
-                           pxEtherHeader->xSourceAddress.ucBytes[ 2 ],
-                           pxEtherHeader->xSourceAddress.ucBytes[ 3 ],
-                           pxEtherHeader->xSourceAddress.ucBytes[ 4 ],
-                           pxEtherHeader->xSourceAddress.ucBytes[ 5 ] ) );
+        if( xHasPrinted == pdFALSE )
```

Comment:
"Bounce back" looks out of context. It would nice to put some context in the message.